### ISSUE TITLE
Inconsistent spacing caused by wrong use of SafeAreView

### ISSUE THREAD

**Issue by noumantahir** [ecency/ecency-mobile#1920]
There are several place in app that uses SafeAreaView to wrap a small component instead of wrapping the screen. This implementation leads to inconsistent spacing on various devices 

SafeAreView should either only be used with full screen wrapper or used in a customised manner to achieve consistency spacing... I personally user iphone-x-helper package for that manner

**Screenshots or Video**
![Screenshot 2021-04-15 at 3 41 35 AM](https://user-images.githubusercontent.com/6298342/114789595-c7f34500-9d9c-11eb-8057-621ddecb050f.png)

**Smartphone:**
 - Device:  iPhone8, iPhone11
 - Version 3.0.17

**Comment by feruzm** [comment]
Yeah, I agree... Consistent design is important otherwise experience will be and engagement with app will be low.

**Comment by noumantahir** [comment]
We extensively working on safe area in our last update, this can now be closed
